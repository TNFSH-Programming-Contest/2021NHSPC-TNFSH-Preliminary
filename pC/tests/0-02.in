9
1 3 
2 1 * 0 
1 3 
3 
2 2 * 0 
1 3 
3 
2 1 + 2 
3 
