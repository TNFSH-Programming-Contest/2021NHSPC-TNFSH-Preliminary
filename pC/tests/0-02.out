3
3
0
0
no
no
